lib/matplotlib/backends/qt_compat.py

```diff
@@ -65,7 +69,7 @@
     except KeyError:
         raise RuntimeError(
             ('Unrecognized environment variable %r, valid values are:'
-             ' %r, %r or %r' % (QT_API_ENV, 'pyqt', 'pyside', 'pyqt5')))
+             ' %r, %r or %r' % (QT_API_ENV, 'pyqt', 'pyside', 'pyqt5', 'pyside2')))
```

Comment:
I also didn't adjust the number of argument placeholders. I fixed both issues. Do I --force an --amend commit or create a new pull request?